shared/src/main/java/com/faforever/neroxis/biomes/BiomeName.java

```diff
@@ -0,0 +1,23 @@
+package com.faforever.neroxis.biomes;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum BiomeName {
+    BRIMSTONE("Brimstone"),
+    DESERT("Desert"),
+    EARLYAUTUMN("EarlyAutumn"),
+    FRITHEN("Frithen"),
+    LOKI("Loki"),
+    MARS("Mars"),
+    MOONLIGHT("Moonlight"),
+    PRAYER("Prayer"),
+    STONES("Stones"),
+    SYRTIS("Syrtis"),
+    WINDINGRIVER("WindingRiver"),
+    WONDER("Wonder");
+
+    private final String value;
```

Comment:
This should probably be named better so that it is understood what it is used for

Reply:
`folderName` maybe?

Reply:
That works